test/clients.go

```diff
@@ -29,7 +29,7 @@ type Context struct {
 // Clients holds instances of interfaces for making requests to various APIs
 type Clients struct {
 	Kube            *kubernetes.Clientset
-	ServingOperator servingoperatorv1alpha1.ServingV1alpha1Interface
+	ServingOperator servingoperatorv1alpha1.OperatorV1alpha1Interface
```

Comment:
In theory, it's only the upgrade tests that need to worry about the old API, so I'm fine with the golang tests assuming the latest one.

Reply:
I think the purpose of the backward compatibility tests is to make sure that our old configs work well with the new release as well, not just through upgrading. @bbrowning wrote it in https://issues.redhat.com/browse/SRVLS-199?focusedCommentId=13941620&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-13941620

Reply:
I'm ok with the spirit of that goal, but in practice, it may be impossible, e.g. when a 1.3 script doesn't fully-qualify the KS CRD name, then it'll break when 1.4 renames the CRD. How flexible are we about editing those old scripts?

Reply:
The idea is to ensure compatibility of older KS CRD (and CR) with newer versions of Serverless, not Bash script compatibility. We can edit/change the Bash scripts as long as we ensure that we still use an old KS CR in some tests that run in CI. 
Anyway, it might be too early to try to keep backwards compatibility with the old configs at this point. I would say we could easily start with GA release of Serverless being the first release that will keep KS CRD (and CR) compatible with future versions.
@bbrowning WDYT?